content/api/relay-webhooks.apib

```diff
@@ -44,56 +43,55 @@ When creating (POST) or updating (PUT) a Relay Webhook:
 **Removing Custom Headers**
 
 When updating (PUT) a Relay Webhook:
+
 ```json
 {
   "custom_headers": {}
 }
 ```
 
-## Field Definitions
+## Relay Webhook Payloads
 
 **SMTP**
 
 The following fields will be included in the JSON object posted to the SMTP relay webhooks target:
 
-| Field     | Type   | Description                                                 | Notes
-|-----------|--------|-----------------------------------------------------------------------|--------------|
-| content   | object | Content that will be used to construct a relay message           | For a full description, see the [Content Attributes](#header-content-attributes). |
-| friendly_from | string | Email address used to compose the "From" header |
-| msg_from | string | [SMTP envelope](http://www.rfcreader.com/#rfc5321_line817) "MAIL FROM", matches "Return-Path" header address |
-| rcpt_to | string | [SMTP envelope](http://www.rfcreader.com/#rfc5321_line817) "RCPT TO" |
-| webhook_id | string | ID of the relay webhook which triggered this relay message |
-| protocol | string | Protocol of the originating inbound message | For smtp payloads, this string will be "smtp" |
++ Data Structure
+    + content (object) - Content that will be used to construct a relay message.
+        + html (string) - Contents of the first text/html part of the message.
+        + text (string) - Contents of the first text/plain part of the message.
```

Comment:
what's the the "first"s here?

Reply:
the message could contain multiple mime parts with the same type (.i.e. `text/html`) we use the first one.